### DISCUSSION TITLE
Customize UI Elements for WebXR / Scene-Viewer / Quick-look

### DISCUSSION THREAD

**Discussion by lupohirp** [google/model-viewer#2203] [Q&A]
Hello,

this is a great project. I'm using it for my platform. 
Is possible to add custom buttons/links element when entering in AR Mode? ( for example, i want to add a  button that add to cart something )

**Comment by elalish** [comment]
Thank you! Using Chrome beta, any child elements you add to model-viewer will show up in webXR. We're planning to flesh this out a bit for the next release. As for the native apps, you can read about our thoughts [here](https://spectrum.chat/model-viewer/general/google-apple-pay-buttons-in-model-viewer~9fc1aeb6-f0d1-4e80-9faf-216c0fed036a).

**Comment by Dovatech** [comment]
Hello,
 I have a same question about  UI (Annotation/Hotspots). I know that Annotation works only with WebXR. Is it possible to insert UI elements in Quick look to? 
Thank you!

**Comment by elalish** [comment]
@Dovatech You'll have to look at QuickLook's documentation. The only thing we pass them is `ios-src`, so it has to be configured there somehow, probably with URL parameters.

**Comment by Dovatech** [comment]
@elalish , thank you !)
